vhost-device-vsock/src/main.rs

```diff
@@ -346,9 +347,7 @@ impl TryFrom<VsockArgs> for Vec<VsockConfig> {
                 _ => cmd_args.param.map_or(Err(CliError::NoArgsProvided), |p| {
                     #[cfg(feature = "backend_vsock")]
                     let backend_info = match (p.uds_path, p.forward_cid) {
-                        (Some(path), None) => {
-                            BackendType::UnixDomainSocket(path.trim().to_string())
-                        }
+                        (Some(path), None) => BackendType::UnixDomainSocket(path.clone()),
```

Comment:
```suggestion
                        (Some(path), None) => BackendType::UnixDomainSocket(path),
```

I'd suggest checking better other instances where you're cloning it, but it's not needed, `-D clippy::redundant_clone` can help on that